scripts/apply_deposits.js

```diff
@@ -1,24 +1,28 @@
-
-/* eslint-disable no-unused-vars */
-/* eslint-disable indent */
 const SnappBase = artifacts.require("SnappBase")
 
 module.exports = async (callback) => {
+  try {
+    const arguments = await process.argv.slice(4)
+    if (arguments.length != 2) {
+      callback("Error: This script requires arguments - <slot> <new state root>")
+    }
+    const [slot, new_state] = arguments
+    
     const instance = await SnappBase.deployed()
-    const [slot, new_state] = await process.argv.slice(4)
-
     const state_index = (await instance.stateIndex.call()).toNumber()
     const curr_state = await instance.stateRoots.call(state_index)
 
     const deposit_state = await instance.deposits.call(slot)
     if (deposit_state.appliedAccountStateIndex != 0) {
-        console.log("Warning: Requested deposit slot has already been applied")
-        callback()
+      callback("Error: Requested deposit slot has already been applied")
     }
 
     console.log("Current slot for :", slot , " with curr_state", curr_state, " and new_state", new_state)
-    const tx = await instance.applyDeposits(slot, curr_state, new_state, deposit_state.shaHash)
+    await instance.applyDeposits(slot, curr_state, new_state, deposit_state.shaHash)
     const deposit_state2 = await instance.deposits.call(slot)
     console.log("New appliedAccountStateIndex is:",deposit_state2.appliedAccountStateIndex )
```

Comment:
Do we not need es-lint disable for the console.log? Did we disable this outside of the file?

Reply:
The es-lint disable that I removed only affected indentation and unused variables, no console.log.

Reply:
Weird. I am surprised to see that my editor is no longer complaining about console.log